src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -72,6 +72,8 @@ If you run a full node, the whole Ethereum network benefits from it.
 
 ## Running your own node {#running-your-own-node}
 
+For general overview on running own Ethereum client instance, refer to [Spin up your own node article](/en/developers/docs/spin-up-your-node/). 
```

Comment:
Interested in running your own Ethereum client? Learn how to [spin up your own node](/en/developers/docs/spin-up-your-node/).